spec/requests/email_domain/delete_spec.rb

```diff
@@ -0,0 +1,17 @@
+describe "DELETE /authorised_email_domains/:id", type: :request do
+  let(:admin_user) { create(:user, super_admin: true) }
+
+  before do
+    https!
+    login_as(admin_user, scope: :user)
+    create(:authorised_email_domain, name: 'some.domain.org.uk')
+  end
+
+  context "when the user is a super admin" do
+    it "deletes the email domain" do
+      expect {
+        delete admin_authorised_email_domains_path(authorised_email_domain)
```

Comment:
Same as above, `authorised_email_domain` doesn't reference anything, so is `nil`.